test/runnable/test16088.d

```diff
@@ -0,0 +1,17 @@
+// REQUIRED_ARGS: -Jrunnable/imports/
```

Comment:
This is a compile issue, no need to run it. Please move to `compilable/`. Runnable tests are a lot slower to autotest than compilable ones.